ytsync/setup.go

```diff
@@ -155,28 +148,31 @@ func (s *Sync) ensureEnoughUTXOs() error {
 	return nil
 }
 
-func (s *Sync) waitUntilUTXOsConfirmed() error {
-	origin := time.Now()
-	for {
-		r, err := s.daemon.UTXOList()
+func (s *Sync) waitForNewBlock() error {
+	status, err := s.daemon.Status()
+	if err != nil {
+		return err
+	}
+
+	for status.Wallet.Blocks == 0 || status.Wallet.BlocksBehind != 0 {
```

Comment:
so is this ok? If there is a problem it would be an infinite loop no? Maybe have a `maxit`?